stellar_contract_sdk/src/testing/mod.rs

```diff
@@ -0,0 +1,8 @@
+#[cfg(not(target_family = "wasm"))]
```

Comment:
These 3 cfg attributes aren't necessary -- the whole testing module is not attached to the crate in that config already.

Reply:
Also this mod declaration is redundant / likely going to break stuff: you're mounting the same file `src/testing/host/mod.rs` at two different paths in the crate: under `crate::host` and `crate::testing::host`. This will produce duplicate definitions of everything inside of it, as you observed with the definition of `impl Debug for Val`.

I'll push a reorg to this branch that I think will get you what you're trying to do here.